airflow/models/dag.py

```diff
@@ -769,23 +769,7 @@ def __init__(
         validate_instance_args(self, DAG_ARGS_EXPECTED_TYPES)
 
     def get_doc_md(self, doc_md: str | None) -> str | None:
```

Comment:
Can we just remove this proxy function?

Reply:
not quite sure. I would like to. but it looks like a public method to me 👀

Reply:
Yeah. better to leave it.

Reply:
Yeah lets keep in that case